.github/workflows/ci.yml

```
@@ -223,7 +222,7 @@ jobs:
         run: /venv/bin/pip install -e .[test] ninja
 
       - name: Test package
-        run: /venv/bin/pytest
+        run: /venv/bin/pytest -n auto
```

```suggestion
        run: /venv/bin/pytest
```

Pytest subprocess the musl lib as an unregistered process. 🙃